### DISCUSSION TITLE
macos15 shortcut keys don't work well

### DISCUSSION THREAD

**Discussion by lid1987** [dbeaver/dbeaver#35718] [Q&A] [answered]
### Description

In the script window, when you use Control+enter to execute a script, a pop-up right-click window will appear. After the right-click window is closed, a window closing pop-up window will pop up

### DBeaver Version

版本: 24.2.2.202409212322

### Operating System

macos Sequoia 15.0

**Comment by Phillipus** [comment, marked as answer]
This is a new feature of macOS Sequoia. See https://www.reddit.com/r/MacOSBeta/comments/1ddvrng/macos_sequoia_beta1_press_ctrl_enter_it_became/

You can disable it or change the key combination in macOS Settings, Keyboard shortcuts.

**Comment by lid1987** [comment]
Thank you！
